gateway/assets/index.html

```diff
@@ -72,7 +72,7 @@ <h1 style="flex: 1 1 auto;">&nbsp; OpenFaaS Portal</h1>
                 <div flex></div>
             </md-content>
 
-            <md-content flex layout="column" ng-repeat="function in functions" ng-show="function.name == selectedFunction.name">
```

Comment:
`ng-show` creates DOM elements and is just hiding it with `css`. If we have tens of functions, the tens of DOM will be created event though only 1 is showing.

Reply:
Looks like a good change.